docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -208,6 +208,7 @@ currently include:
  * `node.kubernetes.io/memory-pressure`: Node has memory pressure.
  * `node.kubernetes.io/disk-pressure`: Node has disk pressure.
  * `node.kubernetes.io/network-unavailable`: Node's network is unavailable.
+ * `node.kubernetes.io/unschedulable`: Node is unschedulable.
```

Comment:
@k82cn would you add version number here as well?

Reply:
ops, done :)